# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/ipdl/message-metadata.ini
# Commit: 2a9873ad6856
# Full Hash: 2a9873ad68561b991cef5795e239a571f08e5aac
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-24 09:31:07
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P4 Remove On[Start|Stop]Request/OnTransportAndData in PHttpChannel, r=mayhemer,nika,necko-reviewers
#   
#   We move OnStartRequest from PHttpChannel to PHttpBackgroundChannel, thus adjusting
#   message-metadata.ini
#   
# ==============================================================================

diff -r be03e0e7b645 -r 2a9873ad6856 ipc/ipdl/message-metadata.ini
--- a/ipc/ipdl/message-metadata.ini	Tue Jun 23 16:56:45 2020 +0000
+++ b/ipc/ipdl/message-metadata.ini	Tue Jun 23 16:56:53 2020 +0000
@@ -12,9 +12,9 @@
 segment_capacity = 8192
 [PContent::StoreAndBroadcastBlobURLRegistration]
 segment_capacity = 8192
-[PHttpChannel::OnStartRequest]
+[PHttpChannel::Redirect1Begin]
 segment_capacity = 8192
-[PHttpChannel::Redirect1Begin]
+[PHttpBackgroundChannel::OnStartRequest]
 segment_capacity = 8192
 [PHttpBackgroundChannel::OnTransportAndData]
 segment_capacity = 8192